1-js/10-error-handling/1-try-catch/article.md

```diff
@@ -1,418 +1,417 @@
+Instrukcja `try...catch` zawiera dwa główne bloki, `try {...}` oraz `catch(err) {...}`:
 
 ```js
 try {
 
-  // code...
+  // instrukcje do wykonania...
 
 } catch (err) {
 
-  // error handling
+  // obsługa błędów z pierwszego bloku...
 
 }
 ```
 
-It works like this:
+Sposób działania:
 
-1. First, the code in `try {...}` is executed.
-2. If there were no errors, then `catch(err)` is ignored: the execution reaches the end of `try` and goes on, skipping `catch`.
-3. If an error occurs, then `try` execution is stopped, and the control flows to the beginning of `catch(err)`. The `err` variable (can use any name for it) will contain an error object with details about what happened.
+1. Rozpoczęcie wykonywania się kodu zawartego w bloku `try {...}`.
+2. Bezbłędne wykonanie pierwszego bloku oznacza, że drugi blok `catch(err) {...}` zostaje pominięty, a program rusza dalej.
+3. W przypadku pojawienia się błędu, wykonywanie bloku `try {...}` zostaje przerwane, a kontrola przekazana jest drugiemu blokowi `catch(err) {...}`. Parametr `err` może mieć dowolną nazwę, argumentem jest obiekt zawierający błąd oraz szczegóły jego wystąpienia.
 
 ![](try-catch-flow.svg)
 
-So, an error inside the `try {…}` block does not kill the script: we have a chance to handle it in `catch`.
+Błąd, który wystąpił w pierwszym bloku `try {...}`, nie przerywa działania programu, więc mamy szansę go obsłużyć w drugim bloku `catch(err) {...}`.
 
-Let's see examples.
+Spójrzmy na przykłady:
 
-- An errorless example: shows `alert` `(1)` and `(2)`:
+- Przykład programu niezawierającego błędu. Wykonuje instrukcje `alert` `(1)` oraz `(2)`:
 
     ```js run
     try {
 
-      alert('Start of try runs');  // *!*(1) <--*/!*
+      alert('początek bloku try');  // *!*(1) <--*/!*
 
-      // ...no errors here
+      // ...instrukcja została wykonana, brak błędu
 
-      alert('End of try runs');   // *!*(2) <--*/!*
+      alert('koniec bloku try');   // *!*(2) <--*/!*
 
     } catch(err) {
 
-      alert('Catch is ignored, because there are no errors'); // (3)
+      alert('blok catch został pominięty, z powodu braku błędu'); // (3)
 
     }
     ```
-- An example with an error: shows `(1)` and `(3)`:
+- Przykład programu zawierającego błąd. Wykonuje instrukcje `(1)` oraz `(3)`:
 
     ```js run
     try {
 
-      alert('Start of try runs');  // *!*(1) <--*/!*
+      alert('instrukcja została wykonana');  // *!*(1) <--*/!*
 
     *!*
-      lalala; // error, variable is not defined!
+      lalala; // niezadeklarowana zmienna powoduje wystąpienie błędu
     */!*
 
-      alert('End of try (never reached)');  // (2)
+      alert('instrukcja nie została wykonana');  // (2)
 
     } catch(err) {
 
-      alert(`Error has occurred!`); // *!*(3) <--*/!*
+      alert(`w programie pojawił się błąd!`); // *!*(3) <--*/!*
 
     }
     ```
 
+````warn header="`try...catch` umożliwia obsługę błędów napotkanych tylko w trakcie wykonywania się programu (ang. Runtime Errors)"
```

Comment:
Proszę umieść to na tej samej linii kodu co angielska wersja, zgodnie z instrukcją w repozytorium